project.clj

```diff
@@ -8,5 +8,5 @@
   :main ^:skip-aot tic-tac-toe.core
   :target-path "target/%s"
   :aliases {"autotest" ["with-profile" "test" "auto" "test"]}
-  :profiles {:test {:plugins [[lein-auto "0.1.2"]]}
+  :profiles {:test {:plugins [[lein-auto "0.1.2"] [cljfmt "0.3.0"]]}
```

Comment:
Generally each plugin vector gets its own line in `project.clj`, much like `refer`. 

Here is an example of a longer `project.clj` https://github.com/technomancy/leiningen/blob/master/sample.project.clj